### ISSUE TITLE
Schema stitching support

### ISSUE THREAD

**Issue by Rajivhost** [fsprojects/FSharp.Data.GraphQL#184]
Hi, 
How about schema stitching support like for example Apollo server/tools does.

https://www.apollographql.com/docs/graphql-tools/schema-stitching.html

https://github.com/hasura/schema-stitching-examples/blob/master/multiple-remote-graphql-apis/index.js

Interesting if we want to make a GraphQL Gateway, I mean something like this:

![image](https://user-images.githubusercontent.com/3539984/47607611-0bfdfe00-da1a-11e8-950e-3177681c1ca4.png)

**Comment by ivelten** [comment]
I did not give a complete though on this yet, but If we do this, I think we should achieve it by a middleware. This feature sounds nice, but I don't think it will we good as a core component - sounds more like an extension for the schema handling.

**Comment by Rajivhost** [comment]
I agree, this can be provided as middleware (why not in the FSharp.Data.GraphQL.Server.Middlewares.AspNetCore project or in an other project name?).
